help/destinations/catalog/streaming/take-action-in-qualtrics.md

```diff
@@ -0,0 +1,176 @@
+
+![URL](/help/destinations/assets/catalog/streaming/take-action-in-qualtrics/json-event-url.png)
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate audiences to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+This destination has an open schema so you may send any properties to Qualtrics.
+
+#### Map attributes
+
+To add an attribute to your mapping, simply select **custom attributes** when adding a new mapping. You may enter any name for your attribute. Qualtrics encourages the *camelCase* naming convention for attribute names (see below screenshot for an example).
+
+![Custom attribute](/help/destinations/assets/catalog/streaming/take-action-in-qualtrics/custom-attribute.png)
+
+See below screenshot for an example of possible attribute mappings.
+
+![Example mappings](/help/destinations/assets/catalog/streaming/take-action-in-qualtrics/example-mappings.png)
+
+#### Map identities
+
+It's mandatory to select an identity namespace for this destination. The two possible source field to target field mappings are:
+
+| Source Field       | Target Field          |
+|--------------------|-----------------------|
+| IdentityMap: Email | Identity: email       |
+| IdentityMap: ECID  | Identity: external_id |
+
+See below screenshot for an example.
+
+![Identity namespace](/help/destinations/assets/catalog/streaming/take-action-in-qualtrics/identity-namespace.png)
+
+## Exported data / Validate data export {#exported-data}
+
+As previously mentioned, this destination uses an open schema, so any properties may be sent to Qualtrics. Nonetheless, the data sent to Qualtrics will follow the below structure:
```

Comment:
One suggestion here: to validate the successful data export, is there let's say any UI screen in Qualtrics where a user can go and check that the profiles or audiences from Adobe Experience Platform have made it through successfully?